TestingTool/task.py

```diff
@@ -16,8 +16,12 @@ class Task(EmbeddedDocument):
     def run(self, browser_instance):
         data = dict()
         result = False
-        result, data = ACTION_LIST[self.action](self.args, browser_instance)
-        if self.reverse_result:
+        try:
```

Comment:
E recomandat sa nu encapsulam in try/except mai mult decat trebuie, ceea ce e si cazul aici. Vrem sa prindem erori doar din executia actiunii, nu si din crearea rezultatului, asa ca ar trebui mutat toata logica 'if result, etc...' in afara except-ului